pkgs/development/tools/misc/lsof/default.nix

```diff
@@ -1,9 +1,12 @@
 { stdenv, fetchurl }:
-
+let pkgs = import <nixpkgs> {};
```

Comment:
pkgs should be added as an argument to the function; otherwise a new and potentially different nixpkgs will be instantiated here